osu.Game/Screens/Edit/Compose/Components/Timeline/TimelineArea.cs

```diff
@@ -55,11 +64,9 @@ private void load()
                                     },
                                     new FillFlowContainer
                                     {
-                                        Anchor = Anchor.CentreLeft,
-                                        Origin = Anchor.CentreLeft,
```

Comment:
Feels better if the settings get centred rather than sticking on the top:
![CleanShot 2021-04-15 at 07 49 39](https://user-images.githubusercontent.com/22781491/114815579-23cec580-9dbf-11eb-9dd1-30f76e3fd544.png)

I guess it was not done that way because it would move away from the cursor position when toggling it?

Reply:
Precisely. See the commit message.